### PR TITLE
CS4009: added note that the Main method can be async

### PR DESCRIPTION
**Pull request by pkulikov**

### CODE REVIEW

File: docs/csharp/misc/CS4009.md [hunk 1]
```diff
@@ -39,6 +39,9 @@ public class Example
 
 ## To correct this error
 
+> [!NOTE]
+> Starting with C# 7.1, the `Main` method can have an `async` modifier. For more information, see the [Async main](../whats-new/csharp-7-1.md#async-main) section of the [What's new in C# 7.1](../whats-new/csharp-7-1.md) article. For the information about how to select C# language version, see the [Select the C# language version](../language-reference/configure-language-version.md) article.
```
Comment (rpetrusha) [inline docs/csharp/misc/CS4009.md]: the information --> information
C# language version --> the C# language version

File: docs/csharp/misc/CS4009.md [hunk 2]
```diff
@@ -39,6 +39,9 @@ public class Example
 
 ## To correct this error
 
+> [!NOTE]
```
Comment (rpetrusha) [inline docs/csharp/misc/CS4009.md]: I think that I would move the note up to the previous section, to follow the "You cannot use the `async` keyword..." sentence. I'd also change it from NOTE to IMPORTANT.

**Review by rpetrusha — CHANGES_REQUESTED** [review]
Some minor comments, and I also thought the note might be moved, @pkulikov. If it is moved, there are two ways to correct the error that can be noted in the **To correct this error** section: the one mentioned, and compiling with C# 7.1 or later.

**Comment by pkulikov** [comment]
@rpetrusha I've addressed your feedback and updated the **To correct this error** section. Please check the updated edition.

**Review by rpetrusha — APPROVED** [review]
Your changes look really good, @pkulikov. Thanks for making the additional ones. I'll merge your PR now.

**Pull request merged**